### PR TITLE
[Remove Immediate] LPS-188559 Remove immediate property from DLProcessorRegistryImpl

### PR DESCRIPTION
**Pull request by rafaprax**
Ticket: https://liferay.atlassian.net/browse/LPS-188559

### CODE REVIEW

**Comment by rafaprax** [comment]
ci:test:relevant

**Comment by rafaprax** [comment]
This change was already validated by QA here https://github.com/liferay-core-infra/liferay-portal/pull/5615. I had to resend it due to code conflicts.

Thx

**Comment by rafaprax** [comment]
just fixed the semver issue.

**Comment by rafaprax** [comment]
Hey @shuyangzhou, Could please review this one here? It is the last PR related to the immediate true. Also, It was ready since last Wed and I had to resend because some code conflicts.

PS: If it pass on your review today, I can check with @catarineaguiar tomorrow morning(pt_BR) and forward to BChan. 

Thanks in advance

**Comment by rafaprax** [comment]
ci:test:sf

**Comment by rafaprax** [comment]
just added the https://github.com/liferay-core-infra/liferay-portal/pull/5689/commits/007ce74cb80b928f47532aedd95df79ffa4f49b6 commit

**Comment by rafaprax** [comment]
ci:test:relevant

**Comment by rafaprax** [comment]
ci:test:sf

**Comment by rafaprax** [comment]
ci:test:relevant

**Comment by rafaprax** [comment]
ci:test:document

**Comment by catarineaguiar** [comment]
no regressions found

File: portal-test/src/com/liferay/portal/test/rule/ExpectedLogs.java [hunk 1]
```diff
@@ -23,6 +23,6 @@
 
 	public String level();
 
-	public Class<?> loggerClass();
```
Comment (shuyangzhou) [inline portal-test/src/com/liferay/portal/test/rule/ExpectedLogs.java:26]: @rafaprax can you still keep this ```public Class<?> loggerClass();``` ? it is still preferred when the Class is accessible from the compiling context for type safety purpose.

**Pull request closed without merging**